mne_bids/dig.py

```diff
@@ -290,6 +291,23 @@ def _coordsystem_json(*, raw, unit, hpi_coord_system, sensor_coord_system,
             'iEEGCoordinateUnits': unit,  # m (MNE), mm, cm , or pixels
         }
 
+    # note that any coordsystem.json file shared within sessions
+    # will be the same across all runs (currently). So
+    # overwrite is set to True always
+    # XXX: improve later when BIDS is updated
+    # check that there already exists a coordsystem.json
+    if datatype == 'meg' and Path(fname).exists() and not overwrite:
```

Comment:
This shouldn't be limited to MEG, right?
```suggestion
    if Path(fname).exists() and not overwrite:
```